v2/internal/frontend/assetserver/common.go

```diff
@@ -8,9 +8,31 @@ import (
 	"net/http"
 	"strings"
 
+	"github.com/wailsapp/wails/v2/pkg/options"
+	"github.com/wailsapp/wails/v2/pkg/options/assetserver"
 	"golang.org/x/net/html"
 )
 
+func IsAssetFsDefined(options *options.App) bool {
+	opts := BuildAssetServerConfig(options)
+	return opts.Assets != nil
+}
+
+func BuildAssetServerConfig(options *options.App) *assetserver.Options {
+	if opts := options.AssetServer; opts != nil {
+		if options.Assets != nil || options.AssetsHandler != nil {
+			panic("It's not possible to use the deprecated Assets and AssetsHandler options and the new AssetServer option at the same time. Please migrate all your Assets options to the AssetServer option.")
```

Comment:
In the case of windows, this panic may not be visible to the developer. That's not a problem as I'm sure it'll be obvious to work it out but as a general thought, I'm wondering if we should have a global recovery which dumps the error to a `panic.txt` and still panics.

Reply:
Yeah, that's a good point always forget about that on Windows. We have a ticket for the global recovery #1613

Reply:
What about adding a MessageBox for windows in this case?

Reply:
Yeah, I think it's ok as is - just a related thought 👍